Classes/Utility/MiscellaneousUtility.php

```diff
@@ -21,8 +22,8 @@
 class MiscellaneousUtility
 {
 
-    /** Overhead used by unique integer generation. Allows 10 billion records before collision */
-    const UNIQUE_INTEGER_OVERHEAD = 10000000000;
```

Comment:
PHP_INT_MAX wouldn't really apply here since now, it's a positive number and the unique integer value would automatically become too large. Now it starts from 18181 and only applies to real colPos numbers, so it should be "safe enough". Knock on wood.